app/src/main/res/menu/list_action_bar.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item
+        android:id="@+id/mode_like"
+        android:title="like" />
```

Comment:
Those title are not very user friendly ;-)

Reply:
done